tools/dos/main.go

```diff
@@ -34,7 +34,7 @@ func initPayload() {
 
 	login.ServerBoundLoginStart{
 		Name:          "Test",
-		HasPublicKey:  false,
+		HasSigData:    false,
```

Comment:
`HasSignature` might be a better name, what do you think?

Reply:
Makes more sense tbh. I was just trying to use what the guys in wiki.vg named it https://wiki.vg/index.php?title=Protocol&oldid=17873#Login_Start

Reply:
I see, in that case this name also makes sense!